apps/sms/test/marionette/lib/views/composer/view.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+/* global module */
+
+var ComposerAccessor = require('./accessors');
+
+function ComposerView(client) {
+  this.client = client;
+  this.accessors = new ComposerAccessor(client);
+}
+
+ComposerView.prototype = {
+  // TODO Move these constants to marionette
+  KEYS: {
+    backspace: '\ue003',
+    enter: '\ue007'
+  },
+
+  get recipients() {
+    var recipients = this.accessors.recipients;
+    return recipients.map(function(recipient) {
+      return recipient.text();
+    });
+  },
+
+  addNewRecipient: function(recipient) {
+    var recipientsList = this.accessors.recipientsList;
```

Comment:
With this method there is a chance that we'll meet [1] (see comments and hackish patch) issue again, if we try to add several recipients consequently.

So either make sure that there are no intermittents on slow configuration this method works on slow configuration, or use the same approach (e.g. at that time I used vagrant to simulate slow env, https://github.com/azasypkin/gaia-vagrant).

Also do we have bug to remove duplicated methods like this from lib/message.js?

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1104961